Doberman/hypervisor.py

```diff
@@ -68,6 +69,15 @@ def setup(self) -> None:
         self.register(obj=self.hypervise, period=self.config['period'], name='hypervise', _no_stop=True)
 
     def shutdown(self) -> None:
+        # shut dow the pipeline monitors
+        for thing in 'alarm control convert'.split():
+            self.run_locally(f"screen -S pl_{thing} -X quit")
+            self.update_config(unactive=f'pl_{thing}')
```

Comment:
Do we want to check it successfully quit (return value from the command) before changing the config, or do we assume it always works?

Reply:
inactive here also rather than unactive?